### ISSUE TITLE
Multiple action types and default case

### ISSUE THREAD

**Issue by cjmyles** [kolodny/redux-create-reducer#13]
Hi, thanks for creating this package! Would you mind updating the README to illustrate how to handle multiple action types (for example you have ADD_TODO, so perhaps REMOVE_TODO as well) so as to clarify the structure of the intended code use? Also, how is it possible to handle the `default` case from a switch statement that your library replaces? Thanks!

**Comment by cjmyles** [comment]
Looking at your code I can see that the default case is handled by returning state:
```
if (handlers.hasOwnProperty(action.type)) {
  return handlers[action.type](state, action);
} else {
  return state;
}
```
In this case, it might be worth updating the README just to illustrate this point?

**Comment by cjmyles** [comment]
Actually, I've just created a PR (social code responsibility and all that)...
https://github.com/kolodny/redux-create-reducer/pull/14

**Comment by kolodny** [comment]
Closed by #14
